src/main/resources/db/changelog/ddl/0028_alter_aed_contact_details_add_last_updated.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+
+    <changeSet author="nathan.russell@valtech.com" id="0028_EIP-5761_aed_contact_details add updated_by and date_updated" context="ddl">
+
+        <addColumn tableName="aed_contact_details">
+            <column name="date_updated" type="timestamp" beforeColumn="version">
```

Comment:
For some reason, I don't think `beforeColumn` works in our liquibase scripts, so you will need to use `afterColumn`. 
If you can run this locally and prove that column is indeed added before version, then that will be appreciated.

Reply:
You are right; `beforeColumn` does not work. I'll try `afterColumn` .... (but I'm not going to lose any sleep over it - I don't think it _really_ matters in what order the columns appear 👍 )

Reply:
OK, I've changed it to `afterColumn`, which results in this 👍 
<img width="1000" alt="Screenshot 2023-04-27 at 19 02 50" src="https://user-images.githubusercontent.com/78348259/234952824-b4341495-dd25-4116-a341-584cbd68dbed.png">